### DISCUSSION TITLE
How to change port from 8080 to others using docker network=host

### DISCUSSION THREAD

**Discussion by nijisakai** [open-webui/open-webui#1559] [Q&A] [answered]
Start server by
`docker run -d --network=host -v open-webui:/app/backend/data -e OLLAMA_BASE_URL=http://127.0.0.1:11434 --name open-webui --restart always ghcr.io/open-webui/open-webui:main
`

**Comment by ElieDeBrauwer** [comment, marked as answer]
Take a look at this page: https://docs.openwebui.com/getting-started/env-configuration 

You can simple pass -e PORT=1234 to make the open-webui accessible on port 1234 instead of 8080.

**Reply by rddvl11** [reply]
Can you pleas eshare the complete command?

**Reply by aryadovoy** [reply]
@rddvl11 
`docker run -d --network=host -v open-webui:/app/backend/data -e OLLAMA_BASE_URL=http://127.0.0.1:11434 -e PORT=1234 --name open-webui --restart always ghcr.io/open-webui/open-webui:main`

**Comment by ManikSinghSarmaal** [comment]
> Start server by `docker run -d --network=host -v open-webui:/app/backend/data -e OLLAMA_BASE_URL=http://127.0.0.1:11434 --name open-webui --restart always ghcr.io/open-webui/open-webui:main `

just add a -p 3000:8080 in your command to set it up at port 3000 or whatever port you desire.
For example `docker run -d -p 3000:8080 -e OLLAMA_BASE_URL=http://127.0.0.1:11434 -v open-webui:/app/backend/data --name open-webui --restart always ghcr.io/open-webui/open-webui:main`

**Comment by raydoomed** [comment]
-p 8080:8080 --net=host

**Reply by tokul** [reply]
> -p 8080:8080 --net=host

-p option does not work in host network mode.
